### PR TITLE
Improves notification panel usability

### LINKED ISSUE #82: Improve dialogues for pending channel openings and deposits
When opening a new channel and depositing the initial deposit into the channel, I am missing a notification that keeps me updated about what is happening. e.g. a pop-up dialogue window on the top right confirming "Opening channel with [target address] has been initiated" and updating me once the channel has been successfully opened and the deposit is in the channel.

Also, it would be nice to get an estimate on how long this might take or at least notify me that it's an on-chain transaction and can take some time.

### LINKED ISSUE #167: Improve notification panel usability
**Is your feature request related to a problem? Please describe.**
At the moment the notification panel is hard to find for new users. Clicking on the identicon is not intuitive for the user.

**Describe the solution you'd like**
Creating an animation that moves the notification pop ups into the identicon would make it clear to the user where they are visible. Creating additional pop up notifications for new pending actions would make it clear that the user can also see the pending actions there.

### PR DESCRIPTION
**Pull request by manuelwedler**
Closes #167 
also closes #82

### CODE REVIEW

**Comment by manuelwedler** [comment]
Also closes #82

File: src/app/app.component.ts [hunk 1]
```diff
@@ -75,6 +76,17 @@ export class AppComponent implements OnInit, OnDestroy {
                 });
             }
         );
+
+        const newNotificationSubscription = this.notificationService.newNotification$.subscribe(
+            () => {
+                this.notificationBlink = 'rgba(255, 255, 255, 0.5)';
+                setTimeout(() => {
```
Comment (andrevmatos) [inline src/app/app.component.ts]: Hmm, `setTimeout` is a little bit ugly, and can give issues later on with zones. Any chance of replacing this with a proper Observables approach?

Comment (manuelwedler) [reply src/app/app.component.ts]: You mean by a `delay` operator for example, right? That should be easy to change. 
We use `setTimeout` also for the polling mechanisms. Would you replace it everywhere?

Comment (manuelwedler) [reply src/app/app.component.ts]: Alright. Thank you :)

**Pull request merged**